packages/ckeditor5-ckbox/src/ckboxui.ts

```diff
@@ -97,4 +86,28 @@ export default class CKBoxUI extends Plugin {
 			} );
 		}
 	}
+
+	/**
+	 * Creates a button for CKBox command to use either in toolbar or in menu bar.
+	 */
+	private _createButton<T extends typeof ButtonView | typeof MenuBarMenuListItemButtonView>( ButtonClass: T ): InstanceType<T> {
+		const editor = this.editor;
+		const locale = editor.locale;
+		const view = new ButtonClass( locale ) as InstanceType<T>;
+		const command = editor.commands.get( 'ckbox' )!;
+		const t = locale.t;
+
+		view.set( {
+			label: t( 'Open file manager' ),
```

Comment:
I'm wondering if we should unify the labels with `CKFinder`.

Reply:
That sounds like a task for another day 🙂